redis/client.py

```diff
@@ -769,6 +770,27 @@ def lastsave(self):
         """
         return self.execute_command('LASTSAVE')
 
+    def migrate(self, host, port, key, destination_db, timeout, copy=False, replace=False, auth=None, keys=[]):
+        """
+        Atomically transfer a key from a Redis instance to another one.
+        migrate(host, port, key, destination_db, timeout, copy=False, replace=False, auth=None, *keys)
+        """
+        remain_args = None
```

Comment:
Other functions in the lib use `args`, and then call something like self.execute('somecommand', *args). Would be nicer to be consistent.